core/templates/dev/head/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -61,138 +60,139 @@
     </ul>
   </ul>
   <ul class="nav oppia-navbar-nav oppia-navbar-profile">
-    <li ng-class="{'open' : $ctrl.activeMenuName === 'profileMenu'}" uib-dropdown ng-if="$ctrl.userIsLoggedIn" class="protractor-test-profile-dropdown">
-      <a uib-dropdown-toggle class="oppia-navbar-dropdown-toggle"
-         ng-mouseover="$ctrl.openSubmenu($event, 'profileMenu')"
-         ng-mouseleave="$ctrl.closeSubmenuIfNotMobile($event)"
-         ng-keydown="$ctrl.onMenuKeypress($event, 'profileMenu', {shiftTab: $ctrl.ACTION_CLOSE, enter: $ctrl.ACTION_OPEN})"
-         role="menuitem"
-         aria-haspopup="true"
-         aria-expanded="false"
-         aria-label="User Menu"
-         tabindex="0">
-        <div class="oppia-navbar-profile-picture-container">
-          <span ng-if="$ctrl.profilePictureDataUrl">
-            <img ng-src="<[$ctrl.profilePictureDataUrl]>" class="oppia-navbar-profile-picture img-circle" alt="User Avatar">
-            <span class="caret"></span>
-          </span>
-          <span ng-if="!$ctrl.profilePictureDataUrl">
-            <i class="material-icons md-40" style="margin-top: -1px;">&#xE853;</i>
-            <span class="oppia-icon-accessibility-label">User Avatar</span>
-            <span class="caret" style="margin-top: -26px;"></span>
-          </span>
-
-          <div class="oppia-navbar-dashboard-indicator ng-cloak" ng-if="$ctrl.numUnseenNotifications > 0">
-            <span class="oppia-navbar-dashboard-indicator-text">
-              <[$ctrl.numUnseenNotifications]>
+    <ul ng-if="!$ctrl.windowIsNarrow" class="nav oppia-navbar-tabs">
```

Comment:
yes, ul in ul is wrong semantically, we were using ul in ul before also in this PR I have updated it to flex design. and it satisfies the need I looked in possibilities of wrapping it into <li> to make it correct semantically but I believe it will be difficult to understand

Reply:
This is still not OK -- make sure things are done correctly semantically, please. It's important for screenreaders, for example.

If needed, please feel free to fix any existing issues with ul in ul, but that is optional for this PR.

Reply:
I've updated the sequence PTAL Thanks!